sme_finance_application_schema/actor_v1

```diff
@@ -19,6 +19,20 @@
             "type": "integer",
             "description": "The outstanding value of mortgages on personal properties belonging to this actor, in GBP"
         },
+        "is_property_residence": {
+            "title": "Is property residence",
+            "description": "Is this property the same as your current home address",
+            "type": "string",
+            "enum": [
+                "no",
+                "yes"
+            ]
+        },
+        "other_properties": {
```

Comment:
Should this not also be an enum (yes / no)? IMO that's what the question suggests.

Reply:
This is work done, before had thats in Master, which was not the same with master, this came about when I merged Master in my branch since I checkout out from Dev.

I want all of the schema to be the same